docs/constants/navigation.js

```diff
@@ -369,6 +369,7 @@ export const general = [
         makePage('guides/in-app-purchases.mdx'),
         makePage('guides/using-push-notifications-services.mdx'),
         makePage('guides/using-feature-flags.mdx'),
+        makePage('guides/using-a-cms.mdx'),
```

Comment:
```suggestion
        makePage('guides/using-cms.mdx'),
```

We should rename this file and URL to `/using-cms` for consistency.

Reply:
I know it's annoying that it's different, but "Using CMS" is not grammatically correct.

grammar < consistency? 🙈 I couldn't decide.